modules/microshift-data-cleaning-full-cleanup.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+//
+// * microshift_troubleshooting/microshift-cleanup-data.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-data-cleaning-full-cleanup_{context}"]
+= Full cleanup of data and configuration
+
+. Run the following command to clean all the {microshift-short} data and configuration.This command performs the following cleanup actions:
+
+* Stop and disable {microshift-short} services
+* Delete all {microshift-short} pods
+* Delete all container image storage
+* Reset network configuration
+* Delete the /var/lib/microshift data directory
+* Delete OVN configuration
+
+.Procedure
+
+. Run the following command to clean all the {microshift-short} data and configuration.
++
+[WARNING]
+====
+This command deletes all {microshift-short} and user-specific data. Use with caution.
+====
++
+[source,terminal]
+----
+$ sudo microshift-cleanup-data --all
+----
++
+[NOTE]
+====
+The command prompts you with a message to confirm the operation. Type 1 or Yes to continue, or anything else to abort the cleanup.
+====
++
+.Example output when you abort the cleanup
+[source,terminal]
+----
+DATA LOSS WARNING: Do you wish to stop and clean ALL MicroShift data AND cri-o container workloads?
+1) Yes
+2) No
+#? no
+Aborting cleanup
+----
++
+.Example output when you continue the cleanup
+[source,terminal]
+----
+DATA LOSS WARNING: Do you wish to stop and clean ALL MicroShift data AND cri-o container workloads?
+1) Yes
+2) No
+#? 1
+Stopping MicroShift services
+Disabling MicroShift services
+Removing MicroShift pods
+Removing crio image storage
+Deleting the br-int interface
+Killing conmon, pause and OVN processes
+Removing MicroShift configuration
+Removing OVN configuration
+MicroShift service was stopped
+MicroShift service was disabled
+Cleanup succeeded
+----
+
+. Run the following command to restart the {microshift-short} service.
```

Comment:
Please, use the full text from the doc about microshift service stopped and disabled. It's important.

Reply:
done.  Note is added